README.md

```diff
@@ -171,6 +174,103 @@ Example usage for the tax year 2024/25:
 cgt-calc --year 2024 --raw sharesight_trxs_dir/
 ```
 
+</details>
+ <br />
+
+### Offshore fund-specific instructions
+
+For correct taxation on [offshore funds](https://www.gov.uk/government/publications/offshore-funds-self-assessment-helpsheet-hs265/hs265-offshore-funds) you need to specify the yearly excess reported income (ERI) from each fund you have owned.
+You can find the full list of funds that requires this at [HMRC](https://www.gov.uk/government/publications/offshore-funds-list-of-reporting-funds).
+
+The tool already includes such yearly history in the [resources folder](https://github.com/KapJI/capital-gains-calculator/blob/main/cgt_calc/resources/eri). You can check if you're fund is already included, or provide a custom ERI history file using the instructions below.
```

Comment:
```suggestion
The tool already includes such yearly history in the [resources folder](https://github.com/KapJI/capital-gains-calculator/blob/main/cgt_calc/resources/eri). You can check if your fund is already included, or provide a custom ERI history file following the instructions [here](https://github.com/KapJI/capital-gains-calculator/blob/main/excess_reported_income_sources.md)). We strongly suggest sharing compiled ERI data so it can be added to the package as it can save significant time to other users that hold the same fund.
```